src/System.Drawing.Common/tests/Printing/PrintDocumentTests.cs

```diff
@@ -257,7 +257,6 @@ private void AssertDefaultPageSettings(PageSettings pageSettings)
             }
 
             Assert.False(pageSettings.Landscape);
```

Comment:
`pageSettings.Landscape` is a customizable setting. Delete?

Reply:
Sure.  As noted in the PR description, I didn't proactively look through the tests for other things that could change based on settings, but might as well delete this one, too, as it's obvious.